index.html

```diff
@@ -15,6 +16,9 @@
     </script>
 </head>
 <body>
+<p>You will be redirected to the documentation below in 5 seconds. Note
```

Comment:
The redirect code does not wait for 5 seconds so this message is a bit misleading.